projects/angular-contenteditable-accessor/src/lib/contenteditable-value-accessor.ts

```diff
@@ -154,6 +154,6 @@ export class ContenteditableValueAccessor
     private static processValue(value: string | null): string {
         const processed = value || '';
 
-        return processed.trim() === '<br>' ? '' : processed;
+        return processed.toString().trim() === '<br>' ? '' : processed;
```

Comment:
The type of coming value above is `string | null`. Moreover `processed` can't be `null` because we cover this case with `value || ''`. According to the code, there is no case to catch `processed.trim is not a function`

I suppose the problem is not here or is not **only** here. Please, could you explain how did you get it?

Reply:
I suppose you've put something other than null / string as initial value of your control? Please share a use case so we can think of optimal way to handle it too.

Reply:
I got that issue too. I used [ngModel] and bind a number

Reply:
I ran into the same issue. I'm using contenteditable spans to adjust some numbers inline. Have to convert them manually to strings in my model to get rid of this issue.

Reply:
Ok, got it. You do realize that after you edit it, it would become a string though, right? I'll merge this PR and release a patch.